projects/observability/src/shared/graphql/model/schema/filter/id/graphql-id-filter.ts

```diff
@@ -16,7 +16,7 @@ export class GraphQlIdFilter implements GraphQlFilter {
         operator: new GraphQlEnumArgument(this.operator),
         value: this.id,
         type: new GraphQlEnumArgument(GraphQlFilterType.Id),
-        idType: new GraphQlEnumArgument(this.idScope)
+        idScope: this.idScope
```

Comment:
I will approve but I am not sure if removing `GraphQlEnumArgument` here would cause any issues.

Reply:
Thanks! No issue. This is the expected type for `idScope`, which replaces `idType`.